tests/test_api_v1.py

```diff
@@ -2458,34 +2458,15 @@ def test_get_content_manifests_by_requests(app, client, db, auth_env, tmpdir):
             ],
         ],
         ["ref=b50b93a32df1c9d700e3e80996845bc2e13be848", 1, ["https://github.com/org/bar.git"]],
-        [
-            "ref=",
-            3,
-            [
-                "https://github.com/org/bar.git",
-                "https://github.com/org/baz.git",
-                "https://github.com/org/foo.git",
-            ],
-        ],
-        ["ref=a-git-ref", None, 'The \\"ref\\" parameter must be a 40 character hex string'],
+        ["ref=a-git-ref", None, "is too short"],
         [
             "pkg_manager=gomod",
             2,
             ["https://github.com/org/foo.git", "https://github.com/org/baz.git"],
         ],
-        [
-            "pkg_manager=",
-            3,
-            [
-                "https://github.com/org/bar.git",
-                "https://github.com/org/baz.git",
-                "https://github.com/org/foo.git",
-            ],
-        ],
```

Comment:
So I take it we lost the ability to specify empty params? But this seems like one of the more OK breaking changes